Degiro conversion error - module not found

Hi everyone. 
I'm trying to convert my order history exported from Degiro but I'm getting the following error:

```
➜  Export-To-Ghostfolio git:(main) npm run start degiro

> export-to-ghostfolio@1.0.0 start
> tsx ./src/manual.ts "degiro"


node:internal/process/esm_loader:34
      internalBinding('errors').triggerUncaughtException(
                                ^
Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/username/Scaricati/Export-To-Ghostfolio/node_modules/dayjs/plugin/customParseFormat' imported from /home/username/Scaricati/Export-To-Ghostfolio/src/converters/degiroConverter.ts
Did you mean to import "dayjs/plugin/customParseFormat.js"?
    at finalizeResolution (node:internal/modules/esm/resolve:265:11)
    at moduleResolve (node:internal/modules/esm/resolve:933:10)
    at defaultResolve (node:internal/modules/esm/resolve:1157:11)
    at nextResolve (node:internal/modules/esm/hooks:866:28)
    at S (file:///home/username/Scaricati/Export-To-Ghostfolio/node_modules/tsx/dist/esm/index.mjs?1714900896692:2:2951)
    at nextResolve (node:internal/modules/esm/hooks:866:28)
    at Hooks.resolve (node:internal/modules/esm/hooks:304:30)
    at handleMessage (node:internal/modules/esm/worker:196:24)
    at Immediate.checkForMessages [as _onImmediate] (node:internal/modules/esm/worker:138:28)
    at process.processImmediate (node:internal/timers:478:21) {
  code: 'ERR_MODULE_NOT_FOUND',
  url: 'file:///home/username/Scaricati/Export-To-Ghostfolio/node_modules/dayjs/plugin/customParseFormat'
}
Node.js v20.12.2
```

Same issue when using Docker image or local build.
I'm on Fedora 40, Node version v20.12.2

--- a/GitVersion.yml
+++ b/GitVersion.yml
@@ -1,4 +1,4 @@
-next-version: 0.12.0
+next-version: 0.12.1
 assembly-informational-format: "{NuGetVersion}"
 mode: ContinuousDeployment
 branches:

--- a/package.json
+++ b/package.json
@@ -16,21 +16,21 @@
   "devDependencies": {
     "@types/cacache": "^17.0.2",
     "@types/jest": "^29.5.11",
-    "@types/node": "^20.11.25",
+    "@types/node": "^20.12.8",
     "jest": "^29.7.0",
     "ts-jest": "^29.1.1",
     "ts-node": "^10.9.2",
-    "tsx": "^4.7.2",
+    "tsx": "4.9.0",
     "typescript": "^5.4.5"
   },
   "dependencies": {
     "@types/cli-progress": "^3.11.5",
-    "cacache": "^18.0.2",
+    "cacache": "^18.0.3",
     "chokidar": "^3.6.0",
     "cli-progress": "^3.12.0",
     "closest-match": "^1.3.3",
     "csv-parse": "^5.5.5",
-    "dayjs": "^1.11.10",
+    "dayjs": "^1.11.11",
     "dotenv": "^16.4.5",
     "yahoo-finance2": "^2.11.2"
   }

